ui/src/templates/constants/defaultTemplates.ts

```diff
@@ -1,2965 +1,16 @@
-          },
-          name: 'Processes',
-        },
-      },
-      {
-        type: 'view',
-        id: '039d8c0b68434000',
-        attributes: {
-          properties: {
-            shape: 'chronograf-v2',
-            queries: [
-              {
-                text:
-                  'from(bucket: v.bucket)\n  |> range(start: v.timeRangeStart)\n  |> filter(fn: (r) => r._measurement == "swap")\n  |> filter(fn: (r) => r._field == "total" or r._field == "used")\n  |> window(period: v.windowPeriod)\n  |> mean()\n  |> group(columns: ["_value", "_time", "_start", "_stop"], mode: "except")\n  |> yield(name: "mean")',
-                editMode: 'advanced',
-                name: '',
-                builderConfig: {
-                  buckets: [],
-                  tags: [{key: '_measurement', values: []}],
-                  functions: [],
-                },
-              },
-            ],
-            axes: {
-              x: {
-                bounds: ['', ''],
-                label: '',
-                prefix: '',
-                suffix: '',
-                base: '10',
-                scale: 'linear',
-              },
-              y: {
-                bounds: ['', ''],
-                label: '',
-                prefix: '',
-                suffix: '',
-                base: '10',
-                scale: 'linear',
-              },
-              y2: {
-                bounds: ['', ''],
-                label: '',
-                prefix: '',
-                suffix: '',
-                base: '10',
-                scale: 'linear',
-              },
-            },
-            type: 'xy',
-            legend: {},
-            geom: 'line',
-            colors: [],
-            note: '',
-            showNoteWhenEmpty: false,
-          },
-          name: 'Swap',
-        },
-      },
-      {
-        id: '0399e8fd61294000',
-        type: 'variable',
-        attributes: {
-          name: 'bucket',
-          arguments: {
-            type: 'query',
-            values: {
-              query: 'buckets()\n  |> map(fn: (r) => r.name)\n',
-              language: 'flux',
-            },
-          },
-          selected: null,
-        },
-      },
-    ],
-  },
-  labels: [],
-})
+export const localMetricsTemplate = () => {
+  return LOCAL_METRICS
+}
 
-export const gettingStartedWithFluxTemplate = () => ({
-  meta: {
-    name: 'Getting Started with Flux',
-    version: '1',
-  },
-  content: {
-    data: {
-      attributes: {
-        description: '',
-        name: 'Getting Started with Flux',
-      },
-      relationships: {
-        cell: {
-          data: [
-            {
-              id: '03afa77a93d82000',
-              type: 'cell',
-            },
-            {
-              id: '03afa77a9b582000',
-              type: 'cell',
-            },
-            {
-              id: '03afa77aa3582000',
-              type: 'cell',
-            },
-            {
-              id: '03afa77aa6182000
```

Comment:
I think these were originally functions returning objects rather than just objects to guard against someone mutating them and causing subtle bugs.

We could preserve the existing behavior by importing the JSON files as text and returning `JSON.parse(theText)` from these functions... or we could not worry about it and assume no one will mutate them down the line. If we do the latter then I think we should update whatever uses these functions to just use the `SYSTEM_TEMPLATE` or whatever constant object directly.